remix-tests/src/compiler.ts

```diff
@@ -105,7 +105,7 @@ export function compileContractSources(sources: SrcIfc, importFileCb: any, opts:
     async.waterfall([
         function loadCompiler (next: Function) {
             compiler = new RemixCompiler(importFileCb)
-            compiler.onInternalCompilerLoaded()
+            compiler.loadVersion(usingWorker, versionUrl)
```

Comment:
don't we need for solcjs to be loaded before continuing?

Reply:
It was working fine on local testing with IDE. Although we should wait for it to be loaded.

Reply:
Now we wait for compiler to load.